pep-9999.rst

```diff
@@ -0,0 +1,861 @@
+been drafted [3_] to propose a common convention across the Scientific Python
+ecosystem for dropping support for older Python versions.
+
+While the exact formulation of that policy is still being discussed, the initial
+proposal is very simple: support any Python feature release published within
+the last 42 months.
+
+For an 18 month feature release cadence, that works out to always supporting at
+least the two most recent feature releases, and then dropping support for all
+X.Y.z releases around 6 months after X.(Y+2).0 is released. This means there is
+a 6 month period roughly every other year where the three most recent feature
+releases are supported.
+
+For a 12 month release cadence, it would work out to always supporting at
+least the three most recent feature releases, and then dropping support for all
+X.Y.z releases around 6 months after X.(Y+3).0 is released. This means that
+for half of each year, the four most recent feature releases would be supported.
+
+For a 24 month release cadence, a 42 month support cycle works out to always
+supporting at least the most recent feature release, and then dropping support
+for all X.Y.z feature releases around 18 months after X.(Y+1).0 is released.
+This means there is a 6 month period every other year where only one feature
+release is supported. Under the proposal in this PEP, that period would
+correspond to the final few rolling beta releases and the release candidate
+phase for the upcoming stable feature release.
+
+
+Release cycle alignment for core development sprints
+----------------------------------------------------
+
+With the proposal in this PEP, it is expected that the focus of core
+development sprints would shift slightly based on the current location
+in the two year cycle.
+
+In release years, the timing of PyCon US is suitable for new contributors to
+work on bug fixes and smaller features before the first release candidate goes
+out, while the Language Summit and core developer discussions can focus on
+plans for the next release series.
+
+The post-release core development sprint in release years will provide an
+opportunity to incorporate feedback received on the release, either as part of
+the next maintenance release (for bug fixes and feedback on provisional APIs),
+or as part of the next release series. These sprints would also likely
+correspond with the Steering Council elections for the next release cycle.
+
+In non-release years, the focus for both events would just be on the upcoming
+maintenance and beta releases. These less intense years would hopefully provide
+an opportunity to tackle various process changes and infrastructure upgrades
+without impacting the release candidate preparation process.
+
+
+Release cycle alignment for prominent Linux distributions
+---------------------------------------------------------
+
+Some rolling release Linux distributions (e.g. Arch, Gentoo) may be in a
```

Comment:
(Not a suggestion, just commentary)

As a present Arch user, I am hoping that they would adopt the proposed rolling release Python model. This would be great for obtaining the latest features.